protos/feast/types/FeatureRow.proto

```diff
@@ -39,4 +39,8 @@ message FeatureRow {
     // <project>/<feature-set-name>:<version>. This value will be used by the feast ingestion job to filter
     // rows, and write the values to the correct tables.
     string feature_set = 6;
+
+    // Identifier tying this feature row to a specific ingestion dataset. For  
+    // batch loads, this dataset id can be attributed to a single ingestion job.
```

Comment:
Is this an optional or required field? How do stream producers interact with it?

Reply:
Optional. Stream producers will not populate this field